Renderer.js

```diff
@@ -0,0 +1,38 @@
+class Renderer {
+  constructor() {
+    this.view = {
+      $boardTemplate: $("#board-template"),
+      $boardContainer: $("#board"),
+      $scoresTemplate: $("#scores-template"),
+      $scoresContainer: $("#scores"),
+      $gameOverTemplate: $("#game-over-template"),
+      $gameOverContainer: $("#game-over-container"),
+    }
```

Comment:
this is a little bit redundant - instead of defining this and then use `const $boardTemplate = this.view.$boardTemplate` for example, just write `$("#board-template")` - it's shorter and less complicated